components/molecule/tabs/src/index.js

```diff
@@ -21,7 +21,9 @@ class MoleculeTabs extends Component {
   _createHandleChange(index) {
     return event => {
       event.preventDefault()
-      if (this.props.items[index].enabled !== false) {
+      const {items, allowHandleDisabledTabClick} = this.props
+      const {enabled} = items[index]
+      if (enabled !== false || allowHandleDisabledTabClick) {
```

Comment:
I think this should be a && instead ||.

Reply:
Instead of `enabled !== false` we could: `enabled === true` that is the same than just `enabled`, right? Or am I missing something?  And yes, it should be `&&` good catch @miduga